arm/Microsoft.Authorization/roleDefinitions/deploy.bicep

```diff
@@ -1,81 +1,85 @@
 targetScope = 'managementGroup'
 
-@description('Required. Name of the custom RBAC role to be created.')
+@sys.description('Required. Name of the custom RBAC role to be created.')
```

Comment:
Please change `roleName` to `name` here and in all other relevant files

Reply:
This is aligning to the doc https://docs.microsoft.com/en-us/azure/templates/microsoft.authorization/roledefinitions?tabs=bicep

Reply:
As our motivation was to stick to the API to use name, I'd suggest to keep `roleName` here

Reply:
@eriqua , thoughts on the above please?

Reply:
Of course fine in that case